docs/visual-basic/language-reference/error-messages/bc30577.md

```diff
@@ -9,21 +9,58 @@ helpviewer_keywords:
 ms.assetid: c2c55640-5c61-4e66-97a4-4322020c6001
 ---
 # 'AddressOf' operand must be the name of a method (without parentheses)
-The `AddressOf` operator creates a procedure delegate instance that references a specific procedure. The syntax is as follows.  
-  
- `AddressOf` `procedurename`  
-  
- You inserted parentheses around the argument following `AddressOf`, where none are needed.  
-  
- **Error ID:** BC30577  
-  
-## To correct this error  
-  
-1. Remove the parentheses around the argument following `AddressOf`.  
-  
-2. Make sure the argument is a method name.  
-  
+
+The `AddressOf` operator creates a procedure delegate instance that references a specific procedure. The syntax is as follows:
+
+```vb
+AddressOf procedurename
+```
+
+You inserted parentheses around the argument following `AddressOf`, where none are needed.
+
+**Error ID:** BC30577
+
+## Example
+
+The following example generates bc30577:
+
+```vb
+Public Sub CountZeroToTen()
+    For i = 0 To 10
+        Console.WriteLine($"Counted: {i}")
+        Threading.Thread.Sleep(500)
+    Next
+End Sub
+
+Sub Main()
+    ' Any of the following two lines generates bc30577.
+    'Dim t As New Threading.Thread(AddressOf(CountZeroToTen))
+    'Dim t As New Threading.Thread(AddressOf CountZeroToTen())
+    t.Start()
+End Sub
+```
+
+## To correct this error
+
+1. Remove the parentheses around the argument following `AddressOf` as shown in the following example:
```

Comment:
I'd like you to rework these changes.

The markdown validation shows a symptom of what's happened. The linter is now parsing the list as two separate numeric lists, with the second starting at '2'. Readers will have the same confusion because the two suggestions are now separated by a block of code. I recommend putting the two numbered items together, and putting the code sample after the list.

Reply:
I think indenting the code snippet *may* fix the issue. I've made that and waiting for the markdown lint to finish to see if it shows errors.

Reply:
@BillWagner, lint error is gone. Does this fix seem good ?